trs_filer/api/additions.openapi.yaml

```diff
@@ -548,6 +548,13 @@ components:
             Other values MAY be used, as long as implementors are aware of the issues discussed in https://tools.ietf.org/html/rfc6920#section-9.4[RFC6920].
 
             GA4GH may provide more explicit guidance for use of non-IANA-registered algorithms in the future.
+    AdditionalType:
```

Comment:
Please rename to `OtherType` because `AdditionalType` might be ambiguous/confusing (like someone would have to add this type _in addition_ to the other types, like descriptor or image).